pages/blog/_.vue

```diff
@@ -3,30 +3,17 @@
     data-qa="blog post"
     class="text-page figure-attribution"
   >
-    <b-container>
-      <b-row class="justify-content-center">
-        <b-col
-          cols="12"
-          class="col-lg-8 pt-large"
-        >
-          <BlogPost
-            :date-published="post.datePublished"
-            :title="post.name"
-            :description="post.description"
-            :body="post.articleBody"
-            :identifier="post.identifier"
-            :hero="hero"
-            :hero-image="heroImage"
-            :authors="post.authorCollection.items.length > 0 ? post.authorCollection.items : null"
-          />
-          <BlogTags
-            v-if="post.keywords"
-            :tags="post.keywords"
-          />
-        </b-col>
-      </b-row>
-      <b-row class="footer-margin" />
-    </b-container>
+    <BlogPost
```

Comment:
For some reason images don't seem to show on blog pages, I don't know if it's due to the img specific styling with the moved css or if it's an issue with the data being passed through in the models here. They do work on exhibition pages, so I think its' due to the models here being passed the wrong attribute at some point.

Reply:
this was mysteriously fixed with the addition of the CTA...